pom.xml

```diff
@@ -99,6 +99,17 @@
                 <cmake.cflags/>
             </properties>
         </profile>
+        <profile>
+            <id>mac-arm-make</id>
+                <activation>
+                    <property>
+                        <name>build_mac_m1</name>
```

Comment:
call it "arm64" not "m1"

$100 says there's an "m2" in the next year or so. They're running out of Pro Max Ultra adjectives.